### PR TITLE
{math,vis}[GCCcore/10.2.0] Octave v6.2.0, GL2PS v1.4.2, Qhull v2020.2, ...

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/q/Qhull/Qhull-2020.2-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ higher dimensions. Qhull implements the Quickhull algorithm for computing the
+ convex hull.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.qhull.org/download/']
+sources = ['%(namelower)s-%(version_major)s-src-8.0.2.tgz']
+#patches = [
```
Comment (verdurin) [inline easybuild/easyconfigs/q/Qhull/Qhull-2020.2-GCCcore-10.2.0.eb:21]: @SimonPinches Please remove the commented-out portions if they're not required.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/q/Qhull/Qhull-2020.2-GCCcore-10.2.0.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/q/Qhull/Qhull-2020.2-GCCcore-10.2.0.eb]: Done - apologies for the delay @verdurin.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (5 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/6c4eb6cc9104bdc8dd90c693c8a88a98 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/q/Qhull/Qhull-2020.2-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d, and
+ higher dimensions. Qhull implements the Quickhull algorithm for computing the
+ convex hull.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.qhull.org/download/']
+sources = ['%(namelower)s-%(version_major)s-src-8.0.2.tgz']
```
Comment (verdurin) [inline easybuild/easyconfigs/q/Qhull/Qhull-2020.2-GCCcore-10.2.0.eb:20]: @SimonPinches please add the missing checksum here.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/q/Qhull/Qhull-2020.2-GCCcore-10.2.0.eb (hunk 2 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/q/Qhull/Qhull-2020.2-GCCcore-10.2.0.eb:20]: @verdurin Done :-)

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**